packages/encryption/src/messageSignature.ts

```diff
@@ -1,13 +1,13 @@
-import { sha256 } from 'sha.js';
-import { encode, decode, encodingLength } from 'varuint-bitcoin';
+import { sha256 } from '@noble/hashes/sha256';
```

Comment:
Is the `sha.js` dep still used? It's still in package.json

Reply:
Nope, was intended to be removed — I missed it, thanks 🙏